### ISSUE TITLE
Difficultés de transcription - Journal de Mme de Genlis

### ISSUE THREAD

**Issue by margauxfre** [HTR-United/CREMMA-MSS-18#15]
Bonjour à tous,

Quelques mots me restent illisibles dans la transcription du journal, je vous les indique ci-dessous : 

- f15 : "un hercule se reposant Sur Sa massue : avec ces [???] = apres les travaux le repos"
![f15](https://user-images.githubusercontent.com/92535159/155702077-eebd05e4-93c1-47b7-b2de-35fac0776dce.png)

- f41 : "d'un ton de couleur Singulier, trop [crud?], mais d'une delicatesse achevée, et d'un detail charmant."
![f41](https://user-images.githubusercontent.com/92535159/155702385-cdf33d17-4a3e-4859-b649-a0f71ddb5dfb.png)

- f59 : "[???] ce 5 Septembre de S^t leu" 
![f59](https://user-images.githubusercontent.com/92535159/155702533-0d546fd2-9bcf-4117-a448-8435c7059566.png)

- f66 : "on nous a fait voir L'original manuscrit des pensées de pascal, écrit de Sa main un très ancien manuscrit Sur velin [???] lettres or et argent, c'est un psautier."
![f66](https://user-images.githubusercontent.com/92535159/155702683-7318061f-7f96-46da-a7f1-806204a6ef3e.png)

Merci :)

**Comment by alix-tz** [comment]
Bonjour,

F15: je lis "mots"

F41: je lis "crud" aussi (et si on pense qu'en anglais c'est "crude color" l'orthographe peut faire sens)

F59: je n'arrive pas à lire quelque chose qui ait du sens. C'est en haut de page?

F66: "violet"?

**Comment by margauxfre** [comment]
Bonjour,

C'est super, merci beaucoup ! je les ajoute dans ma transcription :)
Pour le f59, oui, c'est bien un haut de page et le début d'une nouvelle date donc pas d'indice avec la page précédente...

**Comment by alix-tz** [comment]
@margauxfre Cette issue est-elle résolue ou bien aviez-vous encore des questions en suspend ?

**Comment by margauxfre** [comment]
Pardon, j'ai laissé cette issue de côté... il reste seulement la question du f.59 mais je ne suis même pas sûre que ce soit un mot. Un gribouillis ? Qu'en pensez-vous ?

**Comment by alix-tz** [comment]
Oui, on peut le traiter comme un gribouillis pour le moment, je ne vois vraiment pas non plus ce que ça pourrait être sinon...
